core/src/avm1/activation.rs

```diff
@@ -2864,6 +2864,7 @@ impl<'a, 'gc, 'gc_context> Activation<'a, 'gc, 'gc_context> {
     }
 
     /// Check if a particular property in the scope chain is defined.
+    #[allow(clippy::wrong_self_convention)]
```

Comment:
Just curious: Why does this instance have to be `allow`ed, and not corrected like the rest?

Reply:
I didn't manage to remove the `mut` here. We can rename this function to not be `is_*`, but I think that's less preferred.